packages/utils/tool-utils/src/snapshotNormalizer.ts

```diff
@@ -153,6 +153,46 @@ export function getNormalizedSnapshot(snapshot: ITree, config?: ISnapshotNormali
     };
 }
 
+function normalizeMatrix(value: ITree): ITree {
```

Comment:
is this normalization pass necessary now that we're comparing with semantic equality?

Reply:
This normalization pass is still necessary to solve issues for snapshot tests running in back compat mode.